### PR TITLE
mgmt/storageservice: DeleteStorageService

### LINKED ISSUE #147: No delete for storage services (aka storage accounts) from sdk.

### PR DESCRIPTION
**Pull request by ahmetalpbalkan**
- Added `DeleteStorageService` (fixes #147)
- Renamed `CreateAsync` to `CreateStorageService` for consistency

@paulmey pls review & merge.

### CODE REVIEW

**Comment by paulmey** [comment]
I only see the rename? which LGTM, but I don't want to merge and falsely close 147

**Comment by ahmetalpbalkan** [comment]
@paulmey are u sure? (i'm on mobile but I can see the method in Changes tab)

File: management/storageservice/client.go [hunk 1]
```diff
@@ -78,6 +78,15 @@ func (s StorageServiceClient) CreateAsync(parameters StorageAccountCreateParamet
 	return s.client.SendAzurePostRequest(azureStorageServiceListURL, data)
 }
 
+func (s StorageServiceClient) DeleteStorageService(serviceName string) (management.OperationID, error) {
```
Comment (paulmey) [inline management/storageservice/client.go]: maybe add some godoc?

**Comment by paulmey** [comment]
Hmmm, maybe I missed it. I see it now. LGTM

File: management/storageservice/client.go (hunk 1 above)
Comment (ahmetalpbalkan) [inline management/storageservice/client.go]: erm, others don't have. :) consistency...

Comment (paulmey) [inline management/storageservice/client.go]: your scale for consistency is inconsistent. other packages do have godoc, so you might as well add it for this one... :P

**Pull request merged**